doc/releases/changelog-dev.md

```diff
@@ -118,6 +118,11 @@
 
 <h3>Breaking changes 💔</h3>
 
+* `GlobalPhase` is considered non-differentiable with tape transforms.
+  As a consequence, `qml.gradients.finite_diff` and `qml.gradients.spsa_grad` no longer
+  support differentiation of `GlobalPhase`.
+  [(#5620)](https://github.com/PennyLaneAI/pennylane/pull/5620) 
```

Comment:
@trbromley This is the downside of this PR. I think we can live with that?

Reply:
:thinking: what causes `finite_diff` to be problematic here but `param_shift` is not?

Reply:
```suggestion
* `GlobalPhase` is considered non-differentiable with tape transforms.
  As a consequence, `qml.gradients.finite_diff` and `qml.gradients.spsa_grad` no longer
  support differentiation of `GlobalPhase` with state-based outputs.
  [(#5620)](https://github.com/PennyLaneAI/pennylane/pull/5620) 
```
So should the changelog read like this?

Reply:
Oh yeah, that is a fair point! Thank you!

Reply:
@trbromley Are we good to merge, then? :)